blockchain/txSigner.go

```diff
@@ -1,41 +1,61 @@
 package blockchain
 
 import (
+	"github.com/ElrondNetwork/elrond-go-core/hashing/keccak"
```

Comment:
do not forget to properly rename the file also

Reply:
done